esmvaltool/preprocessor/_time_area.py

```diff
@@ -1,13 +1,13 @@
-########################################################################
-# TIME AND AREA OPERATIONS
-# V.Predoi, University of Reading, May 2017
-########################################################################
+"""Bunch of utilities"""
+from datetime import timedelta as td
```

Comment:
Don't rename `timedelta` to something hard to read, just keep the name `timedelta`.

Reply:
good point; done